ios/SecuritySdk/Classes/CryptographicKey.swift

```diff
@@ -6,14 +6,12 @@
 import Foundation
 
 protocol CryptographicKey {
-    static var algorithm: SecKeyAlgorithm { get }
-    
     /**
-     Initializer that wraps an instance of `SecKey`
+     Initializer that wraps an instance of `SecKey` and `SecKeyAlgorithm`
      
      - throws: an error type `CryptographicKeyError`
      */
-    init(secKey: SecKey) throws
+    init(_ secKey: SecKey, _ algorithm: SecKeyAlgorithm) throws
```

Comment:
swift question: what does this do?

Reply:
removes the need for labels when calling the function